README.md

```diff
@@ -1,29 +1,25 @@
-# Adobe I/O CodeLabs template
+# Build Event-Driven Firefly App Using Custom Event
 
-## Getting started 
+### Overview
 
-Click on the "Use this template" button to copy the template to your namespace. 
+[Project Firefly](https://github.com/AdobeDocs/project-firefly) is a complete framework that enables enterprise developers to build and deploy custom web applications that extend Adobe Experience Cloud solutions and run on Adobe infrastructure. It leverages modern technologies (JAM stack, serverless computing, Node, and React) and ensures best practices when building applications (event-driven architecture, microservices, continuous integration, and delivery). 
 
-The codelab entry point is the [README](README.md). You can put your codelab content including assets under the [lessons](/lessons) directory. The codelab [manifest.json](manifest.json) holds the navigation information and more. Link your lessons in that file in order to render the navigation.
+[Custom Event](https://github.com/adobe/aio-lib-events) is an open-source SDK for the use of third party customers as part of Project firefly. Adobe I/O custom events allows you to build reactive, event-driven applications, based on events originating from various Adobe services. Events are triggered by event providers and can be listened to by journalling or by registering a webhook, The events SDK provides a wrapper over these API calls making it easier for developers to use it as part of their apps. 
 
-## Preview codelab locally
+### User Story
+There are times when users may wish to register their own apps as event provider and integrate other technologies in an event-driven manner (e.g. with other 3rd-party ESPs, notifying call-centres, conducting look-ups to CRM platforms such as Dynamics prior to an e-mail send, etc) 
+A specific demonstration to illustrate this use case has therefore been put together in this lab:
+* Jane build an app using project firefly and she would like to track the likes been clicked by reader
+* Jane registered her app as event provider using custom event (I/O event library) 
+* Joey could subscribed Events from Jane's cutom app event registration 
+* When reader click like or leave comment, Jane and Joe will all receive this event
 
-The template codelab is based on the Project Helix https://www.project-helix.io/. Follow instructions as explained.
+In this lab, you will learn how to:
+* Build a simple web app using project firefly framework
+* Enable event-driven workflows using Adobe I/O event library with ease 
 
-`npm i -g @adobe/helix-cli && hlx up` is enough to get you started. Make sure to have at least `npm@6.14.x` installed and an initial commit. 
+For your convenience, we provide a complete solution of this codelab at [here](https://github.com/AdobeDocs/adobeio-codelab-customevent-demo)
```

Comment:
I propose to rename it to adobeio-samples-custom-events wdyt ? cc @duynguyen

Reply:
yes +1